docs/NNNN-n-tuple-omit-prefix-storage-layout.md

```diff
@@ -0,0 +1,209 @@
+| urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66 | `66a9/c002/6e8bc430-9c3a-11d9-9669-0800200c9a66` |
+| abc123 | `321c/ba00/abc123`
+
+#### Storage Hierarchy
+
+```
+[storage_root]/
+├── 0=ocfl_1.0
+├── ocfl_layout.json
+├── extensions/
+│   └── NNNN-n-tuple-omit-prefix-storage-layout/
+│       └── config.json
+├── 6927/
+│   └── 8821/
+│       └── 12887296/
+│           ├── 0=ocfl_object_1.0
+│           ├── inventory.json
+│           ├── inventory.json.sha512
+│           └── v1 [...]
+├── 66a9/
+│   └── c002/
+│       └── 6e8bc430-9c3a-11d9-9669-0800200c9a66/
+│           ├── 0=ocfl_object_1.0
+│           ├── inventory.json
+│           ├── inventory.json.sha512
+│           └── v1 [...]
+└── 321c/
+    └── ba00/
+        └── abc123/
+            ├── 0=ocfl_object_1.0
+            ├── inventory.json
+            ├── inventory.json.sha512
+            └── v1 [...]
+            
+```
+
+### Example 2
+
+This example demonstrates mappings where the multi-character delimiter is found one or more times in the OCFL object
+identifier, with default `tupleSize`, `numberOfTuples`, and `reverseObjectRoot`, and modified `zeroPadding`.
+
+#### Parameters
+
+There is no default configuration; therefore, configuration parameters must be provided.
+
+```json
+{
+    "extensionName": "NNNN-n-tuple-omit-prefix-storage-layout",
+    "delimiter": "edu/",
+    "tupleSize": 3,
+    "numberOfTuples": 3,
+    "zeroPadding": "right",
+    "reverseObjectRoot": false
+}
+```
+
+#### Mappings
+
+| Object ID | Object Root Path |
+| --- | --- |
+| https://institution.edu/3448793 | `344/879/300/3448793` |
+| https://institution.edu/abc/edu/f8.05v | `f8./05v/000/f8.05v` |
+
+#### Storage Hierarchy
+
+```
+[storage_root]/
+├── 0=ocfl_1.0
+├── ocfl_layout.json
+├── extensions/
+│   └── NNNN-n-tuple-omit-prefix-storage-layout/
+│       └── config.json
+├── 344/
+│   └── 879/
+│       └── 300/
+│           └── 3448793/
+│               ├── 0=ocfl_object_1.0
+│               ├── inventory.json
+│               ├── inventory.json.sha512
+│               └── v1 [...]
+└── f8./
+    └── 05v/
+        └── 000/
+            └── f8.05v/
+                ├── 0=ocfl_object_1.0
+                ├── inventory.json
+                ├── inventory.json.sha512
+                └── v1 [...]
+```
+
+### Example 3
+
+This example demonstrates mappings that produce directory names that are invalid on unix filesystems; therefore this
+layout cannot be used in a repository that needs to be able to store objects with identifiers like these.
+
+#### Parameters
+
+There is no default configuration; therefore, configuration parameters must be provided.
+
+```json
+{
+    "extensionName": "NNNN-n-tuple-omit-prefix-storage-layout",
+    "delimiter": "info:"
+}
+```
+
+#### Mappings
+
+| Object ID | Object Root Path |
+| --- | --- |
+| info:fedora/object-01 | `fedora/object-01` |
+| https://example.org/info:/12345/x54xz321/s3/f8.05v | `/12345/x54xz321/s3/f8.05v` |
```

Comment:
I see the problem now, I will update this example

Reply:
That's fine. Just noting that what those examples draw attention to is that if the omit-prefix id contains a filesystem path separator, then it would be impossible to create an appropriate object root path, even though a valid filesystem path could be constructed.

Reply:
Agreed, I noticed that, the chars are fine on unix, just not acceptable for making proper path